src/util.js

```diff
@@ -311,18 +311,22 @@ if (typeof Symbol !== "undefined" && Symbol.iterator) {
     asArray = function(v) {
         if (es5.isArray(v)) {
             return v;
-        } else if (v != null && typeof v[Symbol.iterator] === "function") {
+        }
+        if (v != null && typeof v[Symbol.iterator] === "function") {
             return ArrayFrom(v);
         }
         return null;
     };
 }
 
 var isNode = typeof process !== "undefined" &&
-        classString(process).toLowerCase() === "[object process]";
+    classString(process).toLowerCase() === "[object process]";
```

Comment:
tab indentation was broken, but I can return it back if any concerns about it :)